python/grass/jupyter/seriesmap.py

```diff
@@ -188,8 +172,7 @@ def add_names(self, names):
 
     def _render_baselayers(self, img):
         """Add collected baselayers to Map instance"""
-        for grass_module, kwargs in self._base_layer_calls:
-            img.run(grass_module, **kwargs)
+        super()._render_baselayers(img)
```

Comment:
You can just remove this method from the subclasses. When the subclass calls this method, it will automatically use the one in the base class.

Reply:
Thank you, this is heading in the right direction, but the code itself is not running, so you really need to get it setup so you can test. For testing the SeriesMap you can use the doc/notebooks/jupyter_tutorial.ipynb, download nc basic [dataset](https://grass.osgeo.org/sampledata/north_carolina/nc_basic_spm_grass7.zip) and run:
```
m = gj.SeriesMap()
m.add_rasters(["elevation", "geology", "landuse"])
m.show()
```